[2022-03-30 15:18:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:61417->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 1 次重试 
[2022-03-30 15:18:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:49775->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 1 次重试 
[2022-03-30 15:18:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:49778->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 2 次重试 
[2022-03-30 15:18:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:49779->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 1 次重试 
[2022-03-30 15:18:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 3 次重试 
[2022-03-30 15:18:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-30 15:18:47] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"弦月睡帽 /market 猫","message_id":-508789143,"message_seq":441396,"message_type":"group","post_type":"message","raw_message":"弦月睡帽 /market 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"弟中弟","level":"","nickname":"普通人阿文","role":"admin","sex":"unknown","title":"群技术支持","user_id":1270672343},"sub_type":"normal","time":1648624708,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:49784->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 停止上报：已达重试上线 
[2022-03-30 15:18:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:49786->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 3 次重试 
[2022-03-30 15:20:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:20:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:20:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:20:58] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8554,"PacketSent":7775,"PacketLost":0,"MessageReceived":651,"MessageSent":114,"LastMessageTime":1648624785,"DisconnectTimes":0,"LostTimes":0}},"time":1648624846}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8554,"PacketSent":7775,"PacketLost":0,"MessageReceived":651,"MessageSent":114,"LastMessageTime":1648624785,"DisconnectTimes":0,"LostTimes":0}},"time":1648624858}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:17] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"9999999","message_id":460970064,"message_seq":455,"message_type":"group","post_type":"message","raw_message":"9999999","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648624864,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:20] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"/market 弦月长睡帽 猫","message_id":-2106553878,"message_seq":456,"message_type":"group","post_type":"message","raw_message":"/market 弦月长睡帽 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648624868,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8556,"PacketSent":7775,"PacketLost":0,"MessageReceived":653,"MessageSent":114,"LastMessageTime":1648624868,"DisconnectTimes":0,"LostTimes":0}},"time":1648624871}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:21:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:21:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:35] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"/market 弦月长睡帽 猫","message_id":-176703108,"message_seq":457,"message_type":"group","post_type":"message","raw_message":"/market 弦月长睡帽 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648624882,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8558,"PacketSent":7776,"PacketLost":0,"MessageReceived":654,"MessageSent":114,"LastMessageTime":1648624882,"DisconnectTimes":0,"LostTimes":0}},"time":1648624884}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-30 15:21:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-30 15:21:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:63575->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 2 次重试 
[2022-03-30 15:21:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:63576->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 2 次重试 
[2022-03-30 15:21:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:21:57] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"/market 弦月长睡帽 猫","message_id":1707648237,"message_seq":458,"message_type":"group","post_type":"message","raw_message":"/market 弦月长睡帽 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648624900,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:57] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8559,"PacketSent":7776,"PacketLost":0,"MessageReceived":655,"MessageSent":114,"LastMessageTime":1648624900,"DisconnectTimes":0,"LostTimes":0}},"time":1648624901}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:21:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:22:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:22:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:23:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:23:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:23:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:23:28] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8580,"PacketSent":7790,"PacketLost":0,"MessageReceived":659,"MessageSent":117,"LastMessageTime":1648624955,"DisconnectTimes":0,"LostTimes":0}},"time":1648624996}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:23:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:23:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:29:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:29:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:29:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:29:58] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8605,"PacketSent":7810,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625386}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:30:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:30:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:30:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:30:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8605,"PacketSent":7810,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625398}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:30:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:30:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:30:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:30:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8605,"PacketSent":7810,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625411}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:30:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:30:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:30:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:30:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8607,"PacketSent":7812,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625424}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:30:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:30:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:30:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:30:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8607,"PacketSent":7812,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625436}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:30:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:30:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:30:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:31:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8608,"PacketSent":7813,"PacketLost":0,"MessageReceived":662,"MessageSent":119,"LastMessageTime":1648625357,"DisconnectTimes":0,"LostTimes":0}},"time":1648625449}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:31:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:31:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:32:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:32:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:32:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:32:48] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:33:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:33:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:33:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:33:53] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8619,"PacketSent":7821,"PacketLost":0,"MessageReceived":664,"MessageSent":120,"LastMessageTime":1648625568,"DisconnectTimes":0,"LostTimes":0}},"time":1648625621}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:33:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:33:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:34:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:34:06] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8621,"PacketSent":7823,"PacketLost":0,"MessageReceived":664,"MessageSent":120,"LastMessageTime":1648625568,"DisconnectTimes":0,"LostTimes":0}},"time":1648625633}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:34:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:34:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:34:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:34:19] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8621,"PacketSent":7823,"PacketLost":0,"MessageReceived":664,"MessageSent":120,"LastMessageTime":1648625568,"DisconnectTimes":0,"LostTimes":0}},"time":1648625646}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:34:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:34:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:34:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:34:31] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8622,"PacketSent":7824,"PacketLost":0,"MessageReceived":664,"MessageSent":120,"LastMessageTime":1648625568,"DisconnectTimes":0,"LostTimes":0}},"time":1648625659}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:34:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:34:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:35:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:35:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:35:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:35:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:35:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:35:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:35:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:35:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8631,"PacketSent":7828,"PacketLost":0,"MessageReceived":667,"MessageSent":122,"LastMessageTime":1648625705,"DisconnectTimes":0,"LostTimes":0}},"time":1648625721}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:35:35] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"/market 弦月花睡帽 猫","message_id":-1406631393,"message_seq":441405,"message_type":"group","post_type":"message","raw_message":"/market 弦月花睡帽 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"Erdbee","level":"","nickname":"不要说话","role":"admin","sex":"unknown","title":"🍐","user_id":921584143},"sub_type":"normal","time":1648625722,"user_id":921584143}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:35:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:35:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:35:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:36:23] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:37:17] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:37:55] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:38:02] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:38:15] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:38:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:38:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:38:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:38:38] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8648,"PacketSent":7835,"PacketLost":0,"MessageReceived":677,"MessageSent":122,"LastMessageTime":1648625889,"DisconnectTimes":0,"LostTimes":0}},"time":1648625906}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:38:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:38:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:38:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:38:51] [WARNING]: 群消息发送失败: 信息为空. 
[2022-03-30 15:39:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:39:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:39:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:39:23] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8650,"PacketSent":7836,"PacketLost":0,"MessageReceived":678,"MessageSent":122,"LastMessageTime":1648625931,"DisconnectTimes":0,"LostTimes":0}},"time":1648625951}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:39:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:39:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:39:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:39:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8651,"PacketSent":7837,"PacketLost":0,"MessageReceived":678,"MessageSent":122,"LastMessageTime":1648625931,"DisconnectTimes":0,"LostTimes":0}},"time":1648625963}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:39:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:39:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:39:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:41:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:41:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:41:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:41:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:41:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:41:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:41:23] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=c3b36fee162928c24887609c344c227d.image,url=https://gchat.qpic.cn/gchatpic_new/1178098043/675683939-2631403571-C3B36FEE162928C24887609C344C227D/0?term=3,subType=0]","message_id":-1036745784,"message_seq":441414,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=c3b36fee162928c24887609c344c227d.image,subType=0]","self_id":3102833903,"sender":{"age":0,"area":"","card":"九龍葉月","level":"","nickname":"Gymnopédie","role":"member","sex":"unknown","title":"","user_id":1178098043},"sub_type":"normal","time":1648626070,"user_id":1178098043}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:41:23] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8666,"PacketSent":7847,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626071}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:41:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:41:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:41:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:41:36] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8667,"PacketSent":7848,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626083}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:41:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:41:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:41:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:41:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8667,"PacketSent":7848,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626096}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:41:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:41:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:41:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:01] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8668,"PacketSent":7849,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626109}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:42:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8668,"PacketSent":7849,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626121}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:42:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:27] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8668,"PacketSent":7849,"PacketLost":0,"MessageReceived":681,"MessageSent":124,"LastMessageTime":1648626070,"DisconnectTimes":0,"LostTimes":0}},"time":1648626134}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:42:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:42:35] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"？","message_id":-1254902946,"message_seq":441415,"message_type":"group","post_type":"message","raw_message":"？","self_id":3102833903,"sender":{"age":0,"area":"","card":"錯觉","level":"","nickname":"Zed","role":"member","sex":"unknown","title":"👮‍♂️","user_id":252919506},"sub_type":"normal","time":1648626142,"user_id":252919506}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:39] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8670,"PacketSent":7850,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626147}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:42:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:42:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8670,"PacketSent":7850,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626159}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:42:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:42:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8671,"PacketSent":7851,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626172}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:43:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:17] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8671,"PacketSent":7851,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626185}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:43:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8672,"PacketSent":7852,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626197}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:43:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:43:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8672,"PacketSent":7852,"PacketLost":0,"MessageReceived":682,"MessageSent":124,"LastMessageTime":1648626142,"DisconnectTimes":0,"LostTimes":0}},"time":1648626210}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:43] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"/market 猫 猫","message_id":-558017902,"message_seq":480,"message_type":"group","post_type":"message","raw_message":"/market 猫 猫","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648626210,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:43:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:43:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:43:55] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8673,"PacketSent":7852,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626223}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:43:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:44:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8674,"PacketSent":7853,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626235}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:44:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:44:21] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8674,"PacketSent":7853,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626248}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:44:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:44:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8676,"PacketSent":7855,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626261}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:44:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:44:46] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8676,"PacketSent":7855,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626273}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:44:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:44:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:44:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:44:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8676,"PacketSent":7855,"PacketLost":0,"MessageReceived":683,"MessageSent":124,"LastMessageTime":1648626211,"DisconnectTimes":0,"LostTimes":0}},"time":1648626286}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:44:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:03] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"...","message_id":742056676,"message_seq":441416,"message_type":"group","post_type":"message","raw_message":"...","self_id":3102833903,"sender":{"age":0,"area":"","card":"汤不理啵","level":"","nickname":"79","role":"owner","sex":"unknown","title":"⑨🐦","user_id":46025579},"sub_type":"normal","time":1648626290,"user_id":46025579}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:11] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8678,"PacketSent":7856,"PacketLost":0,"MessageReceived":684,"MessageSent":124,"LastMessageTime":1648626290,"DisconnectTimes":0,"LostTimes":0}},"time":1648626299}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:24] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8678,"PacketSent":7856,"PacketLost":0,"MessageReceived":684,"MessageSent":124,"LastMessageTime":1648626290,"DisconnectTimes":0,"LostTimes":0}},"time":1648626311}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:36] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"母爱变质","message_id":1530778226,"message_seq":441417,"message_type":"group","post_type":"message","raw_message":"母爱变质","self_id":3102833903,"sender":{"age":0,"area":"","card":"錯觉","level":"","nickname":"Zed","role":"member","sex":"unknown","title":"👮‍♂️","user_id":252919506},"sub_type":"normal","time":1648626324,"user_id":252919506}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:37] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8680,"PacketSent":7857,"PacketLost":0,"MessageReceived":685,"MessageSent":124,"LastMessageTime":1648626324,"DisconnectTimes":0,"LostTimes":0}},"time":1648626324}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:37] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=8634df7994d3e5ee0028f33366da7a26.image,url=https://gchat.qpic.cn/gchatpic_new/598682694/675683939-2704207525-8634DF7994D3E5EE0028F33366DA7A26/0?term=3,subType=1]","message_id":-880619549,"message_seq":441418,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=8634df7994d3e5ee0028f33366da7a26.image,subType=1]","self_id":3102833903,"sender":{"age":0,"area":"","card":"小黑猪","level":"","nickname":"FrontEnd.","role":"member","sex":"unknown","title":"","user_id":598682694},"sub_type":"normal","time":1648626324,"user_id":598682694}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:48] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"[CQ:image,file=b7f605a0cd4efc3bf9157fc7ab9fc240.image,url=https://gchat.qpic.cn/gchatpic_new/921584143/675683939-2377343700-B7F605A0CD4EFC3BF9157FC7AB9FC240/0?term=3,subType=1]","message_id":-1132069003,"message_seq":441419,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=b7f605a0cd4efc3bf9157fc7ab9fc240.image,subType=1]","self_id":3102833903,"sender":{"age":0,"area":"","card":"Erdbee","level":"","nickname":"不要说话","role":"admin","sex":"unknown","title":"🍐","user_id":921584143},"sub_type":"normal","time":1648626335,"user_id":921584143}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8682,"PacketSent":7857,"PacketLost":0,"MessageReceived":687,"MessageSent":124,"LastMessageTime":1648626335,"DisconnectTimes":0,"LostTimes":0}},"time":1648626337}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:45:54] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"？","message_id":-294382574,"message_seq":441420,"message_type":"group","post_type":"message","raw_message":"？","self_id":3102833903,"sender":{"age":0,"area":"","card":"兔","level":"","nickname":"森本草食","role":"admin","sex":"unknown","title":"🐰","user_id":407152018},"sub_type":"normal","time":1648626341,"user_id":407152018}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:45:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:45:58] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"孝心变质","message_id":-1720507260,"message_seq":441421,"message_type":"group","post_type":"message","raw_message":"孝心变质","self_id":3102833903,"sender":{"age":0,"area":"","card":"兔","level":"","nickname":"森本草食","role":"admin","sex":"unknown","title":"🐰","user_id":407152018},"sub_type":"normal","time":1648626346,"user_id":407152018}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:45:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:46:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8685,"PacketSent":7858,"PacketLost":0,"MessageReceived":689,"MessageSent":124,"LastMessageTime":1648626346,"DisconnectTimes":0,"LostTimes":0}},"time":1648626349}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:46:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:46:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:46:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:49:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:49:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:49:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:49:48] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8727,"PacketSent":7880,"PacketLost":0,"MessageReceived":701,"MessageSent":132,"LastMessageTime":1648626549,"DisconnectTimes":0,"LostTimes":0}},"time":1648626576}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:49:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:49:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:49:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:01] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8728,"PacketSent":7881,"PacketLost":0,"MessageReceived":701,"MessageSent":132,"LastMessageTime":1648626549,"DisconnectTimes":0,"LostTimes":0}},"time":1648626588}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:14] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8728,"PacketSent":7881,"PacketLost":0,"MessageReceived":701,"MessageSent":132,"LastMessageTime":1648626549,"DisconnectTimes":0,"LostTimes":0}},"time":1648626601}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:17] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"有没有肥肥的","message_id":1880129969,"message_seq":441427,"message_type":"group","post_type":"message","raw_message":"有没有肥肥的","self_id":3102833903,"sender":{"age":0,"area":"","card":"想冬眠（猝）","level":"","nickname":"伊","role":"member","sex":"unknown","title":"🐍","user_id":1251230387},"sub_type":"normal","time":1648626604,"user_id":1251230387}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:22] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"来点食材","message_id":-525362144,"message_seq":441428,"message_type":"group","post_type":"message","raw_message":"来点食材","self_id":3102833903,"sender":{"age":0,"area":"","card":"想冬眠（猝）","level":"","nickname":"伊","role":"member","sex":"unknown","title":"🐍","user_id":1251230387},"sub_type":"normal","time":1648626610,"user_id":1251230387}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:26] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8730,"PacketSent":7881,"PacketLost":0,"MessageReceived":703,"MessageSent":132,"LastMessageTime":1648626610,"DisconnectTimes":0,"LostTimes":0}},"time":1648626614}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:39] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8731,"PacketSent":7882,"PacketLost":0,"MessageReceived":703,"MessageSent":132,"LastMessageTime":1648626610,"DisconnectTimes":0,"LostTimes":0}},"time":1648626626}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:50:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:50:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:50:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:50:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:52:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:52:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:55:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:55:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:59:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:59:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:59:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:59:38] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8770,"PacketSent":7912,"PacketLost":0,"MessageReceived":709,"MessageSent":135,"LastMessageTime":1648626949,"DisconnectTimes":0,"LostTimes":0}},"time":1648627166}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:59:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:59:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 15:59:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 15:59:51] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8770,"PacketSent":7912,"PacketLost":0,"MessageReceived":709,"MessageSent":135,"LastMessageTime":1648626949,"DisconnectTimes":0,"LostTimes":0}},"time":1648627178}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 15:59:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 15:59:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:00:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:00:04] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8771,"PacketSent":7913,"PacketLost":0,"MessageReceived":709,"MessageSent":135,"LastMessageTime":1648626949,"DisconnectTimes":0,"LostTimes":0}},"time":1648627191}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:00:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:00:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:00:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:01:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:01:03] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8778,"PacketSent":7918,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627251}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:01:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:01:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:01:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:01:16] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8778,"PacketSent":7918,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627263}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:01:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:01:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:01:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:01:29] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8778,"PacketSent":7918,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627276}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:01:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:01:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:01:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:01:41] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8779,"PacketSent":7919,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627289}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:01:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:01:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:01:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:01:54] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8779,"PacketSent":7919,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627301}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:01:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:02:07] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8781,"PacketSent":7921,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627314}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:02:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:02:19] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8781,"PacketSent":7921,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627327}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:02:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:02:32] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8782,"PacketSent":7922,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627339}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:02:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:02:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:02:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:02:45] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8782,"PacketSent":7922,"PacketLost":0,"MessageReceived":710,"MessageSent":136,"LastMessageTime":1648627211,"DisconnectTimes":0,"LostTimes":0}},"time":1648627352}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:02:46] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"那来点御姐   我通吃","message_id":-268246710,"message_seq":441434,"message_type":"group","post_type":"message","raw_message":"那来点御姐   我通吃","self_id":3102833903,"sender":{"age":0,"area":"","card":"想冬眠（猝）","level":"","nickname":"伊","role":"member","sex":"unknown","title":"🐍","user_id":1251230387},"sub_type":"normal","time":1648627353,"user_id":1251230387}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:02:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:02:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:03:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:08:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:09:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:10:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:11:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:13:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:14:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:14:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:14:08] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8868,"PacketSent":7975,"PacketLost":0,"MessageReceived":734,"MessageSent":145,"LastMessageTime":1648627992,"DisconnectTimes":0,"LostTimes":0}},"time":1648628036}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:14:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:14:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:14:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:15:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:15:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:16:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:16:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:16:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:16:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:16:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:16:33] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8896,"PacketSent":7993,"PacketLost":0,"MessageReceived":740,"MessageSent":149,"LastMessageTime":1648628159,"DisconnectTimes":0,"LostTimes":0}},"time":1648628181}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:16:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:16:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:17:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:18:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:18:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:18:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:18:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:19:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:19:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:19:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:19:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:19:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:19:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8935,"PacketSent":8015,"PacketLost":0,"MessageReceived":752,"MessageSent":154,"LastMessageTime":1648628369,"DisconnectTimes":0,"LostTimes":0}},"time":1648628371}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:19:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:19:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:19:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:19:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:19:52] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":675683939,"message":"我只给诗人","message_id":-1343354137,"message_seq":441458,"message_type":"group","post_type":"message","raw_message":"我只给诗人","self_id":3102833903,"sender":{"age":0,"area":"","card":"玛勒巴子","level":"","nickname":"Somnus","role":"member","sex":"unknown","title":"qc","user_id":2122241781},"sub_type":"normal","time":1648628380,"user_id":2122241781}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:19:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:19:56] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8936,"PacketSent":8015,"PacketLost":0,"MessageReceived":753,"MessageSent":154,"LastMessageTime":1648628380,"DisconnectTimes":0,"LostTimes":0}},"time":1648628383}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:19:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:20:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:20:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:20:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:20:09] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8937,"PacketSent":8016,"PacketLost":0,"MessageReceived":753,"MessageSent":154,"LastMessageTime":1648628380,"DisconnectTimes":0,"LostTimes":0}},"time":1648628396}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:29:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:29:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:29:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-30 16:29:43] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":8981,"PacketSent":8046,"PacketLost":0,"MessageReceived":764,"MessageSent":157,"LastMessageTime":1648628937,"DisconnectTimes":0,"LostTimes":0}},"time":1648628971}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-30 16:29:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:29:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:32:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-30 16:32:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-30 16:32:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
